src/components/MenuItem.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+
+export class MenuItem extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      bread: "Standard",
+      isHot: false,
+      hotPrice: 0.2,
+      total: this.props.price
+    };
+  }
+
+  addBread(event) {
+    let breadName = this.props.breads[event.target.value].name;
+    let breadPrice = this.props.breads[event.target.value].price;
+    let total = this.props.price + breadPrice;
+
+    this.setState({
+      bread: breadName,
+      total: !this.state.isHot ? total : total + this.state.hotPrice
+    });
+  }
+
+  addHot() {
+    const hotPrice = this.state.hotPrice;
+    let isHot = this.state.isHot ? false : true;
```

Comment:
I think this is unnecessary as its going to be `true` or `false` anyway

Reply:
What comes back from the checkbox? Isn't that true or false?

Reply:
wow sure, I simply need to check `is:checked` or whatever it is 👍